File: HexChat/pmcolor.py
__module_name__ = "PMColor"
__module_author__ = "TingPing"
__module_version__ = "1"
__module_description__ = "Color PM tabs like Hilights"

import xchat

def pm_cb(word, word_eol, userdata):
  xchat.command('GUI COLOR 3')
	return None

xchat.hook_print("Private Message to Dialog", pm_cb)
xchat.hook_print("Private Action to Dialog", pm_cb)

File: HexChat/pmcolor.lua
-- SPDX-License-Identifier: MIT
hexchat.register('PMColor', '1', 'Color PM tabs like highlights')

for _, event in pairs({'Private Message to Dialog', 'Private Action to Dialog'}) do
	hexchat.hook_print(event, function (args)
		hexchat.command('gui color 3')
	end)
end
